atlassian_jwt_auth/frameworks/common/exceptions.py

```diff
@@ -1,6 +1,2 @@
 class NoTokenProvidedError(Exception):
     pass
```

Comment:
Should `NoTokenProvidedError` extend `ValueError`?

Reply:
I think inheriting from `Exception` is fine even though we probably should have a common base exception (in a separate PR) that all of our exceptions inherit from.

Reply:
Well arguably our existing exceptions inherit from `ValueError` (https://github.com/atlassian/asap-authentication-python/blob/master/atlassian_jwt_auth/exceptions.py)